packages/edit-site/src/components/sidebar-navigation-screen-main/index.js

```diff
@@ -31,35 +31,40 @@ export default function SidebarNavigationScreenMain() {
 		<SidebarNavigationScreen
 			isRoot
 			title={ __( 'Design' ) }
+			description={ __(
+				'Customise your Navigation menus, Templates, and more.'
```

Comment:
should this be "and Styles" instead of "and more"?

Reply:
Yes.